src/index.html

```diff
@@ -9,6 +9,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
+    <form action="#"
+    method="post"
+    data-qa="big">
```

Comment:
fix indents in all places, should be like this
```suggestion
    <form
      action="#"
      method="post"
      data-qa="big"
    >
```